applications/CompressiblePotentialFlowApplication/python_scripts/level_set_remeshing_process.py

```diff
@@ -46,7 +46,8 @@ def __init__(self, Model, settings ):
                     "distance_threshold"                     : 0.001,
                     "check_at_each_time_step"                : true,
                     "avoid_almost_empty_elements"            : true,
-                    "deactivate_full_negative_elements"      : true
+                    "deactivate_full_negative_elements"      : true,
+                    "variables_list"                         : ["VELOCITY_POTENTIAL","AUXILIARY_VELOCITY_POTENTIAL"]
```

Comment:
This change should be in another PR probably, but since the change in the DistModProcess "breaks" the tests in the PotentialFlowApp (since VELOCITY/PRESSURE are not added as DOFs), I'd rather do this change  here